vendor/k8s.io/system-validators/validators/types_unix.go

```diff
@@ -48,11 +49,14 @@ var DefaultSysSpec = SysSpec{
 			{Name: "PROC_FS"},
 			{Name: "NETFILTER_XT_TARGET_REDIRECT", Aliases: []string{"IP_NF_TARGET_REDIRECT"}},
 			{Name: "NETFILTER_XT_MATCH_COMMENT"},
+			{Name: "FAIR_GROUP_SCHED"},
 		},
 		Optional: []KernelConfig{
 			{Name: "OVERLAY_FS", Aliases: []string{"OVERLAYFS_FS"}, Description: "Required for overlayfs."},
 			{Name: "AUFS_FS", Description: "Required for aufs."},
 			{Name: "BLK_DEV_DM", Description: "Required for devicemapper."},
+			{Name: "CFS_BANDWIDTH", Description: "Required for CPU quota."},
+			{Name: "CGROUP_HUGETLB", Description: "Required for hugetlb cgroup."},
```

Comment:
what does "Required for ..." mean in an `Optional` list?

Reply:
can be interpreted as required kernel options for optional k8s features. no preference on the wording in the description on my side.
cc @KentaTada